twitter/endpoint.bal

```diff
@@ -403,6 +403,30 @@ public isolated client class  Client {
         return check handleStatusArrayResponse(httpResponse);
     }
 
+    # Get a user's tweets for a specific user.
+    # 
+    # + handleName - Handle name of user 
+    # + return - If success, returns 'Tweet' object array, else returns error
+    @display {label: "Get User's Tweets Of Specific User"} 
+    isolated remote function getUserTweets(@display {label: "Handle Name"} string handleName) 
+                                    returns @tainted @display {label: "Array Of Tweet"} Tweet[]|error {
```

Comment:
`@tainted` can be removed.
The use of @tainted is removed from Ballerina. It doesn't do anything

Reply:
Removed